libraries/AP_Common/Bitmask.h

```diff
@@ -65,7 +72,16 @@ class Bitmask {
         }
         return true;
     }
-    
+
+    // return number of bits set
+    uint16_t count() {
+        uint16_t sum = 0;
+        for (uint16_t i=0; i<numwords; i++) {
+            sum += bits[i];
```

Comment:
Am I missing something? I don't see this counting the number of bits set, rather it just appears to be adding all the words up, which is not equal to the number of bits. (IE 0x2 + 0x2 will give you 0x4, even though only two bits are set).